benchmarks/mempool.py

```diff
@@ -162,20 +178,76 @@ async def add_spend_bundles(spend_bundles: List[SpendBundle]) -> None:
                 tasks.append(asyncio.create_task(add_spend_bundles(spend_bundles[peer])))
             await asyncio.gather(*tasks)
             stop = monotonic()
-        print(f"add_spendbundle time: {stop - start:0.4f}s")
-        print(f"{(stop - start) / total_bundles * 1000:0.2f}ms per add_spendbundle() call")
+        print(f"  time: {stop - start:0.4f}s")
+        print(f"  per call: {(stop - start) / total_bundles * 1000:0.2f}ms")
 
-        with enable_profiler(True, f"create-{suffix}"):
+        print("\nProfiling add_spend_bundle() with replace-by-fee")
+        total_bundles = 0
+        tasks = []
+        with enable_profiler(True, f"replace-{suffix}"):
```

Comment:
We seem to be calling `enable_profiler` with `profile` always as `True`. Maybe we could simplify by just removing it?

Reply:
I want to leave the door open to disable the profiler too, as it impacts the performance of the code